deploy/helm/sumologic/conf/logs/otelcol/config.yaml

```diff
@@ -605,7 +603,7 @@ service:
         - resource/containers_copy_node_to_host
         - sumologic_schema
         - source/containers
-        - logstransform/containers_parse_json
```

Comment:
Shouldn't we add it to `logs/fluent/containers` as well?

Reply:
We should, will fix.